data/instruct_advanced_postgres.csv

```diff
@@ -1,16 +1,16 @@
+broker,instructions_cte_window,"What is the ticker symbol, month, average closing price, highest price, lowest price, and MoMC for each ticker each month?",MoMC = month-over-month change in average closing price = (avg_close_given_month - avg_close_previous_month) / avg_close_previous_month for each ticker each month,"WITH monthly_price_stats AS (SELECT DATE_TRUNC('month', sbDpDate) AS MONTH, sbDpTickerId, AVG(sbDpClose) AS avg_close, MAX(sbDpHigh) AS max_high, MIN(sbDpLow) AS min_low FROM sbDailyPrice GROUP BY MONTH, sbDpTickerId) SELECT t.sbTickerSymbol, mps.month, mps.avg_close, mps.max_high, mps.min_low, (mps.avg_close - LAG(mps.avg_close) OVER (PARTITION BY mps.sbDpTickerId ORDER BY mps.month)) / LAG(mps.avg_close) OVER (PARTITION BY mps.sbDpTickerId ORDER BY mps.month) AS mom_change FROM monthly_price_stats mps JOIN sbTicker t ON mps.sbDpTickerId = t.sbTickerId;WITH monthly_prices AS (SELECT sbDp.sbDpTickerId, DATE_TRUNC('month', sbDp.sbDpDate) AS MONTH, AVG(sbDp.sbDpClose) AS avg_close, MAX(sbDp.sbDpHigh) AS max_high, MIN(sbDp.sbDpLow) AS min_low FROM sbDailyPrice sbDp GROUP BY sbDp.sbDpTickerId, DATE_TRUNC('month', sbDp.sbDpDate)), monthly_prices_with_lag AS (SELECT mp.sbDpTickerId, mp.month, mp.avg_close, mp.max_high, mp.min_low, LAG(mp.avg_close) OVER (PARTITION BY mp.sbDpTickerId ORDER BY mp.month) AS prev_avg_close FROM monthly_prices mp) SELECT mpwl.sbDpTickerId, mpwl.month, mpwl.avg_close, mpwl.max_high, mpwl.min_low, (mpwl.avg_close - mpwl.prev_avg_close) / NULLIF(mpwl.prev_avg_close, 0) AS mom_change FROM monthly_prices_with_lag mpwl JOIN sbTicker t ON mpwl.sbDpTickerId = t.sbTickerId ORDER BY mpwl.sbDpTickerId, mpwl.month;","NPM (Net Profit Margin) = (Total Income from Sales - (Tax + Commission Expenses)) / Total Income from Sales * 100, for recent transactions. TAC = Total Active Customers who have recently joined ACP = Average Closing Price of tickers over a recent period MoMC = month-over-month change in average closing price = (avg_close_given_month - avg_close_previous_month) / avg_close_previous_month for each ticker each month"
```

Comment:
The question here requests for the ticker symbol explicitly - the added query returns only the tickerId but not the symbol. Shall we just keep the first query (which returns the symbol)?

Reply:
Got it, will fix -- thanks!